zeebe-cluster/templates/configmap.yaml

```diff
@@ -22,43 +22,8 @@ data:
         ZEEBE_CONTACT_POINTS="${ZEEBE_CONTACT_POINTS},${HOSTNAME::-1}$i.$(hostname -d):26502"
     done
     export ZEEBE_CONTACT_POINTS="${ZEEBE_CONTACT_POINTS}"
+    ls -al /exporters/
+    cp -a /exporters/*.jar /usr/local/zeebe/lib/
```

Comment:
I would prefer to copy the exporters into a separate folder (e.g. `/usr/local/zeebe/exporters/`) to avoid classpath issues.